lib/improver/cli/__init__.py

```diff
@@ -75,6 +75,10 @@ def docutilize(obj):
         doc = getdoc(obj)
     doc = str(NumpyDocstring(doc))
     doc = str(GoogleDocstring(doc))
+    doc = doc.replace(':exc:', '')
+    doc = doc.replace(':keyword', ':param')
```

Comment:
The commit message and PR description explain the first replace of `:exc:` to empty string. Could you also explain the reason for the `keyword` and `kwtype` replacements?

Reply:
"Because @TomekTrzeciak told me to"!
These come from a [notebook](https://github.com/MetOffice/improver_aux/blob/e11b72886e5d5e276b268bba4721035f23c2c451/notebooks/tickets/IMPRO-1333/clize_testing.ipynb) that Tomek put together to test Clize things. I think they allow alternative forms of keyword-argument sections in doc-strings.

Reply:
Allowing other section types in docstrings makes sense and this is a reasonable opportunity to include that change. I'm happy with this now.